### ISSUE TITLE
Paypal on DropIn (v3.23.0 and v4.9.0): pop-up keep loading forever

### ISSUE THREAD

**Issue by thiagomgd** [Adyen/adyen-web#1675]
**Describe the bug**
We already use the DropIn component for card payments, and are adding Paypal to it. 
Followed the steps here: https://docs.adyen.com/payment-methods/paypal/web-drop-in 

When testing, I change the payment to paypal, I click on the button, a popup opens but I just keeps loading.

There's no console error on either our checkout page or the popup. On our page, I see requests to `sandbox.paypal` and they seem successful. There are no network requests at all on the popup.

Below are the API permissions on https://www.sandbox.paypal.com/ 
![image](https://user-images.githubusercontent.com/12170252/182924646-263d7a3b-ef84-49b5-8367-dfa8ba7ba7ee.png)

Our DropIn code:

```
const paymentContainer = React.useRef(null);

    const [dropin, setDropin] = useState<DropinElement | null>(null);

    useEffect(() => {
        const checkout = new AdyenCheckout({
            paymentMethodsResponse,
            locale: 'en-US',
            environment,
            clientKey,
            amount,
            onError,
            onChange,
            paymentMethodsConfiguration,
            showPayButton
        });

        if (paymentContainer.current) {
            const createdDropin = checkout
                .create('dropin')
                .mount(paymentContainer.current || '#spg-adyen-dropin');
            onDropInCreated?.(createdDropin);
            setDropin(createdDropin);
        }
    }, []); // run it once

    return (
            <div id="spg-adyen-dropin" ref={paymentContainer} />
    );
```

and our paymentMethodsConfiguration:

```
const CARD_PAYMENT: CardElementProps = {
    hasHolderName: true,
    holderNameRequired: true,
    enableStoreDetails: false,
    hideCVC: false,
    name: 'Credit card',
    showPayButton: false
};

const PAYPAL_PAYMENT: PayPalElementProps = {
    debug: true,
    showPayButton: true
};

const PAYMENT_METHODS_CONFIGURATION = {
    card: CARD_PAYMENT,
    paypal: PAYPAL_PAYMENT
};
```

EDIT: Also tried updating to V4.9.0, but problem persists and I don't get any errors on `onError` or any other `onXXX` method.

**Comment by ribeiroguilherme** [comment]
Hi @thiagomgd ,

Thanks for reaching out. By checking your implementation, the code looks correct. It seems that the error can be related to the PayPal configuration on the Customer Area or Sandbox. 

Please reach out to our support team so they can guide you to configure your PayPal settings. Hope this helps!

**Comment by thiagomgd** [comment]
I was able to get it to work. Adyen support (through email/zendesk) told me code is NOT correct, and Paypal does not work with custom button, and to make it work I would need to:

1. Call submit;
2. Get response and pass `action` back to the drop-in
3. When user confirms, it fires `onAdditionalDetails`, which needs in the backend to call `payment/details`

Unfortunately, the documentation on integrating Paypal on an existing DropIn component doesn't mention this at all, only another link that support shared with me.

Link I was following: https://docs.adyen.com/payment-methods/paypal/web-drop-in

Link they shared: https://docs.adyen.com/online-payments/web-drop-in/integrated-before-5-0-0 

Seems the first one is not valid for version < 5, which I think should be mentioned in the link, and give a link to the correct one. Specially since the first link has the text below:

> The PayPal Smart Payment Buttons are available from Drop-in V 3.7.0 and later.

This makes it seem as I could follow that guide for the versions I tried
